watcher

```diff
@@ -40,15 +40,11 @@ def run(network):
 
     while True:
         time.sleep(60)
-        try:
-            if latest_deployed_release_commit_has_changed(
-                    network, current_release_commit):
-                logging.info(
-                    "New release has been published. Restarting nearup")
-                restart_nearup(network)
-                current_release_commit = latest_deployed_release_commit(network)
-        except Exception as ex:
-            logging.error(ex)
+        if latest_deployed_release_commit_has_changed(network,
+                                                      current_release_commit):
+            logging.info("New release has been published. Restarting nearup")
+            restart_nearup(network)
```

Comment:
While we're here, I do find the `nearup` term to be a bit confusing here. Shouldn't it be `restart_neard`? Since `nearup` itself doesn't run continuously (other than this watcher of course)?

Reply:
Yes, probabaly we want to remove this in the future, currently nearup restarts both watcher and neard in some cases, maybe the better way would be to explicitly call either stop_neard and stop_watcher

Reply:
Ok, we can circle back